Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -182,7 +182,12 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     // Log share initiated event
     [self userCompletedAction:BNCShareInitiatedEvent];
     UIActivityItemProvider *itemProvider = [self getBranchActivityItemWithLinkProperties:linkProperties];
-    NSMutableArray *items = [NSMutableArray arrayWithObject:itemProvider];
+    NSMutableArray *items;
+    if (itemProvider) {
```

Comment:
I'm closing this PR since this doesn't seem to solve anything.   We can re-open this in the future if needed.